diff --git a/CMakeLists.txt b/CMakeLists.txt
index 911f0ee..38179e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,17 +71,7 @@ include(cmake/Sanitizers.cmake)
 
 # threads
 find_package(Threads REQUIRED)
-
-# open62541
-option(UAPP_INTERNAL_OPEN62541 "Use internal open62541 library" ON)
-
-set(UA_MULTITHREADING 100 CACHE STRING "") # enable thread-safe API (100-199)
-if(UAPP_INTERNAL_OPEN62541)
-    set(OPEN62541_VERSION v1.2.3)  # set manually for CI
-    add_subdirectory(3rdparty/open62541) # target open62541::open62541
-else()
-    find_package(open62541 REQUIRED)
-endif()
+find_package(open62541 REQUIRED)
 
 # open62541pp library
 file(GLOB_RECURSE sources "src/*.cpp")
